### PR TITLE
i18n + English spelling typos + French l10n

### PR DESCRIPTION
**Pull request by gfrenoy**
Some "Loading" messages were still "hard coded" and could not be localized + some typos in English translation.

And now also included : the French translation !

### CODE REVIEW

File: src/config/locales/fr/translation.json [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+      "remove_association": "Supprimer l'assocication",
+      "remove_all_associations": "Supprimer toutes les associations",
+      "confirm": "Confirmer",
+      "cancel": "Annuler",
+      "confirm_message": "Êtes-vous certain de vouloir supprimer toutes les associations ?"
+    },
+    "language": {
+      "title": "Langue",
+      "display_language": "Langue",
+      "notice": "Nécessite un redémarrage",
```
Comment (h3poteto) [inline src/config/locales/fr/translation.json]: __[Misspell]__

"marrage" is a misspelling of "marriage"

File: src/config/locales/fr/translation.json [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    "drop_message": "Déposer pour téléverser sur Mastodon"
+  },
+  "message": {
+    "account_load_error": "Erreur au chargement des comptes",
+    "account_remove_error": "Erreur à la suppression du compte",
+    "preferences_load_error": "Erreur au chargement des préférences",
+    "timeline_fetch_error": "Erreur au chargement du fil public",
+    "notification_fetch_error": "Erreur au chargement des notifications",
+    "favourite_fetch_error": "Erreur au chargement des favoris",
+    "start_streaming_error": "Erreur au démarrage du flux",
```
Comment (h3poteto) [inline src/config/locales/fr/translation.json]: __[Misspell]__

"marrage" is a misspelling of "marriage"

File: src/config/locales/fr/translation.json (hunk 1 above)
Comment (gfrenoy) [reply src/config/locales/fr/translation.json]: I guess it's a bot that made that suggestion ?  In any case, "redémarrage" is the correct French word for "restart".  "Marriage" means "wedding" ;-p

File: src/config/locales/fr/translation.json (hunk 2 above)
Comment (gfrenoy) [reply src/config/locales/fr/translation.json]: Same as above.  "Démarrage" = "Start"

File: src/config/locales/fr/translation.json (hunk 1 above)
Comment (h3poteto) [reply src/config/locales/fr/translation.json]: Oh, sorry. It is a comment from auto linter. And may be it does not consider French. Please ignore the misspelling comments.

**Comment by h3poteto** [comment]
I remove `misspelling` from Sider.

**Pull request merged**